Horion/Command/Commands/EnchantCommand.cpp

```diff
@@ -1,9 +1,44 @@
 #include "EnchantCommand.h"
+#include <map>
 
-
+std::map<std::string, int> enchantMap;
 
 EnchantCommand::EnchantCommand() : IMCCommand("enchant", "Enchants items", "<enchantment> [level]")
 {
+	enchantMap["Protection"] = 0;
+	enchantMap["Fire_Protection"] = 1;
+	enchantMap["Feather_Falling"] = 2;
+	enchantMap["Blast_Protection"] = 3;
+	enchantMap["Projectile_Protection"] = 4;
```

Comment:
Please keep it consistent with uppercase & lowercase

Reply:
fixed